src/index.ts

```diff
@@ -2,5 +2,5 @@ import { api } from './api';
 import { SummaryByProject } from './models';
 
 export const client = (token: string, workspace: string) => ({
-  getSummaryGroupByClients: (sinceDate: string, untilDate: string): Promise<SummaryByProject[]> => api(token, workspace).summaryByProjects(sinceDate, untilDate)
+  getSummaryGroupByClients: (sinceDate: string, untilDate: string): Promise<SummaryByProject[]> => api(token, workspace).summaryByProjects(sinceDate)
```

Comment:
Quítalo también de la definición:
```suggestion
  getSummaryGroupByClients: (sinceDate: string): Promise<SummaryByProject[]> => api(token, workspace).summaryByProjects(sinceDate)
```

Reply:
done!